docs/UserGuide.md

```diff
@@ -166,13 +166,17 @@ The features are broken down to into:
 
 The following table gives an overview of what each parameter used in the Features section is for.
 
-| Parameter    | Description                                                          |
-|--------------|----------------------------------------------------------------------|
-| NAME         | The full name of the doctor/patient                                  |
-| NRIC         | The National Registration Identity Card number of the doctor/patient |
-| PHONE_NUMBER | The phone number of the doctor/patient                               |
-| INDEX        | The position of the appointment/doctor/patient in the filtered list  |
-
+| Parameter    | Description                                                          | Valid Input Note                                                                                                |
```

Comment:
Perhaps use another term instead of "Note", ie. Valid Parameter Input

Reply:
Ok, thanks